pyproject.toml

```diff
@@ -19,6 +19,7 @@ requires = [
 
 [tool.black]
 line-length = 88
+target_version = ['py37', 'py38', 'py39']
```

Comment:
Is there a way to get that from `setup.py` metadata?

Reply:
I think the idea is `pyproject.toml` to be static, to avoid all issues with dynamic behavior in `setup.py`. It's indeed slightly not ideal to change this for each Python version, but if you read https://github.com/psf/black/issues/751 in the end as long as py37 is in there, the rest has no effect. There is no changes in formatting for Python3.7+.